packages/shared/src/utils/property-priorities.js

```diff
@@ -589,7 +589,7 @@ longHandLogical.add('math-style');
 // CSS Pointer Events
 longHandLogical.add('touch-action');
 
-const PRIORITIES = {
+export const PSEUDO_CLASS_PRIORITIES: { [key: string]: number } = {
```

Comment:
these types seem very hacky, but for some reason flow is unable to parse the types of string literals as object keys. I tried to create separate types of unions of literals that are in each of these priority objects, but then I would have trouble passing `prop` to the object, because `string` is not compatible with the union. do you have any other ideas on how to better approach this?

Reply:
Flow requires manual annotations for exports.

So technically, you'd need to copy the object and use it as a type as well. Also, always remember to wrap objects in `$ReadOnly`.